e2e-testing/cypress/functions/CollectionFunctions.js

```diff
@@ -0,0 +1,96 @@
+import CollectionLocators from '../locators/CollectionLocators';
+
+class CollectionFunctions {
+
+    launch_collection_modal(collectionURL) {
+        cy.visit(collectionURL);
```

Comment:
@shakirandagire Let's launch the collections modal through the interface, not directly using a URL
So login, and then navigate into the collections by clicking on the Collections button.

Reply:
@AmTryingMyBest Thanks for the feedback, let me do that